scripts/install-chef-suse.sh

```diff
@@ -295,6 +295,22 @@ ensure_service_running () {
     fi
 }
 
+ensure_crowbar_running ()
+{
+    local crowbar_up=0
+    local error="Errno::ECONNREFUSED"
+    for ((x=1; x<10; x++)); do
+        ($CROWBAR crowbar list --port 3000) 2>&1 >/dev/null | grep -q "$error" || {
+            crowbar_up=1
+            break
+        }
+        sleep 2
+    done
+    if [ -z crowbar_up ]; then
```

Comment:
`-z` doesn't work if you set the initial value to `0`, I think. Also missing `$`.